### ISSUE TITLE
Error "server should be running" when attaching volumes

### ISSUE THREAD

**Issue by vitaliy-zinchenko** [terraform-providers/terraform-provider-scaleway#90]
### Terraform Version
Terraform v0.11.7
provider.scaleway v1.5.1

### Affected Resource(s)
scaleway_volume_attachment
scaleway_server

### Terraform Configuration Files
```hcl
provider "scaleway" {
  organization = ""
  token = ""
  region = "ams1"
}

resource "scaleway_ip" "ip" {
  server = "${scaleway_server.test.id}"
}

resource "scaleway_server" "test" {
  name = "test"
  image = "ca9a9340-92e8-4c5f-8ae1-423466f7ef30"
  type = "C2S"
  depends_on = [
    "scaleway_volume.test1",
    "scaleway_volume.test2",
    "scaleway_volume.test3",
    "scaleway_volume.test4"]
}


resource "scaleway_volume" "test1" {
  name = "test1"
  size_in_gb = 50
  type = "l_ssd"
}

resource "scaleway_volume_attachment" "test1" {
  server = "${scaleway_server.test.id}"
  volume = "${scaleway_volume.test1.id}"
}

resource "scaleway_volume" "test2" {
  name = "test2"
  size_in_gb = 50
  type = "l_ssd"
}

resource "scaleway_volume_attachment" "test2" {
  server = "${scaleway_server.test.id}"
  volume = "${scaleway_volume.test2.id}"
}

resource "scaleway_volume" "test3" {
  name = "test3"
  size_in_gb = 50
  type = "l_ssd"
}

resource "scaleway_volume_attachment" "test3" {
  server = "${scaleway_server.test.id}"
  volume = "${scaleway_volume.test3.id}"
}

resource "scaleway_volume" "test4" {
  name = "test4"
  size_in_gb = 50
  type = "l_ssd"
}

resource "scaleway_volume_attachment" "test4" {
  server = "${scaleway_server.test.id}"
  volume = "${scaleway_volume.test4.id}"
}
```

### Debug Output
https://gist.github.com/vitaliy-zinchenko/c42a6d22b91a61c304c025e8b7ad9128

### Expected Behavior
All 4 volumes attached to the server

### Actual Behavior
Exceptions:
* scaleway_volume_attachment.test2: StatusCode: 400, Type: invalid_request_error, APIMessage: server should be running
* scaleway_volume_attachment.test1: StatusCode: 400, Type: invalid_request_error, APIMessage: server should be running

/cc @nicolai86

**Comment by nicolai86** [comment]
Thanks for the bugreport, @vitaliy-zinchenko ! This too should be fixed once #87 is merged. If you could provide feedback using a compiled version of the branch it would mean a lot!

**Comment by nicolai86** [comment]
this will be fixed with the release of 1.6.0.

**Comment by nicolai86** [comment]
1.6.0 was released
